src/pages/api/thirdparty/graphql.tsx

```diff
@@ -0,0 +1,87 @@
+import { ApolloServer } from '@apollo/server'
+import { DateTime } from 'luxon'
+import { startServerAndCreateNextHandler } from '@as-integrations/next'
+import { GraphQLJSON } from 'graphql-type-json'
+import { GraphQLError } from 'graphql'
+import KeystoneAPI from '../dataSources/keystone'
+
+/* Type Definitions */
+
+/* Data we will return to any third-party querying the API. */
+const typeDefs = `
```

Comment:
It's a bit repetitive to define these typeDefs when these are already defined in the Keystone CMS GraphQL API. We could potentially import the entire schema from Keystone and use that for typeDefs, but then we'd have to manually update it every time a change was made in Keystone, and also would be limited to returning only the data that Keystone exposes.